packages/office-ui-fabric-react/src/utilities/grid/GridCell.Props.ts

```diff
@@ -0,0 +1,62 @@
+export interface IGridCellProps {
+
+  /**
+   * The option that will be made available to the user
+   */
+  item: any;
```

Comment:
The item is really a props file created by the consumer (in SwatchColorPicker it is the IColorCellProps). What is in that props file could vary from any different consumer so I don't think there is anything common between the items. I think your comment about ensuring that item is only passed into the callbacks is the appropriate course of actions, because we won't need type safety